### ISSUE TITLE
[EN][Spell][Hunter] Feign Death

### ISSUE THREAD

**Issue by Batranelu** [Atlantiss/BugTracker#8808]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Feign death doesn't get you out of combat. I even dismissed my pet before using it and still didn't worked. Here's a screenshot: http://imgur.com/a/rjBmk

**Current behaviour**: You remain in combat.

**Expected behaviour**: You should get you out of combat.

**Steps to reproduce the problem**:

1. Get in combat
2. Use Feign Death
3. 
Spell link: http://cata.openwow.com/spell=5384
Proof: https://www.youtube.com/watch?v=X7CmPItwKx4
**Revision**: 5292

**Comment by szapol** [comment]
the video u linked as a proof clearly shows fd doesn't drop combat, it just erases you from the threat table xd no fd is not supposed to get you out of combat

**Comment by TheRealestMcCoy** [comment]
It will only drop combat if you are the last one alive. In the pic you linked you are in ZF you trying to FD while the group members are alive. This will only drop threat. To drop combat you must be the only on alive this means no pet out or pet dead all other party/raid members are dead.

**Comment by fluessig** [comment]
right now feign death literally never drops combat, even if you are the only target in combat with the training dummy

i think the whole "feign death doesnt drop combat"-thing only applies to instanced PVE
outside of that feign death always drops combat

https://www.youtube.com/watch?v=S9RaC2QopFk (1:56)

**Comment by Batranelu** [comment]
Acces this link: http://wowwiki.wikia.com/wiki/Feign_Death    and look at Notes, I quote:
"Using feign death exits the hunter from combat, allowing for such things as eating, mounting, or other activities not possible while in combat. This can also be helpful if needing to afk for a short duration, due to the fact that passing mobs will act as if the hunter is dead."

**Comment by Jenovas** [comment]
Batranelu - that is not true, Feign Death is a very complicated mechanic, FD is not a combat remover only a threat reseter. If you are soloing there is a high chance that you will lose combat (if your pet is fighting / is alive it wont drop combat). When in a party / raid it also doesnt make you leave combat, if it would hunters could use multiple potions during a boss fight. In PvP I think it should remove Combat if noone with combat is interacting with you (for example your healer that has combat heals you / buffs you when FD is active and that might make you stay in combat). I think that in PvE FD works as it should, the problem is that in PvP we have general problems with combat (like if you heal anyone from the team that is in combat you will be in combat for as long as that target is in combat any many other cases).

**Comment by Batranelu** [comment]
@Jenovas if that's the problem, that hunters would use multiple potions during boss fight you can put that as a restriction on FD I've seen it before and this is the way it should work, FD should drop you out of combat since your threat is reduced to 0, but it also shouldn't work in favor of hunters like you said, no multiple potions usage or anything like that. I'm completely sure that FD should get you out of combat and your pet, other players in your paty/raid should remain in combat.
My solution for this , let FD get you out of combat like it should , but put a restriction to NOT RESET POTIONS, cause I've seen this working before on other cataclysm servers.

**Comment by xolon** [comment]
There's nothing wrong with Feign Death on Atlantiss in terms of getting you out of combat because it should not get you out of combat. For closure

**Comment by VgqGHTNm** [comment]
It does get you out of combat when the threat table is completely reset to nill as a result of your FD. It does and always has done this on retail. Trust me, I played Hunter for years on retail, throughout all chapters up to and including Cataclysm, after which I quit.

If the threat reset of FD causes no one else to have any remaining threat, the mob should obviously reset (since there is no one to attack). This in turn should end combat. Since no one is attacking us anymore, because, well the mob reset. This does not happen on Atlantiss. It should.

**Comment by xolon** [comment]
I agree.
But my point is exactly what you wrote.
FD doesn't get you out of combat, it resets your thread. This if there is no one fighting beside you make your enemies reset & run back which leads to dropping out of combat.

**Comment by xolon** [comment]
Ok so, I've been doing some testing.
There is a 30% if mob is right next to you and attacking you that FD will somehow fail, you will get feign death but the mob will just sit there and break it with next attack. I couldn't determine what causes it, bug happened while fding 0.2 s before the attack and 0.2 after mob's attack

**Comment by VgqGHTNm** [comment]
I have noticed that as well. And regarding the other point, I quote you:

> This if there is no one fighting beside you make your enemies reset & run back which leads to dropping out of combat.

Just so we're all clear on this -- this does not currently happen on Atlantiss. Not immediately anyhow. I have noticed that sometimes, if you just remain FD-ed long enough it can happen. Just not with any certainty. So, either way, FD is not working as expected. One of many things about the Hunter that is not working as expected.

**Comment by vanter1282** [comment]
It resets your threat table and when you move the mob that was attacking you will regain all the aggro it had towards you, but you now are on CD.... and it does not matter if you dismiss your pet before or not. You regain threat.

**Comment by CrypticWizard** [comment]
Yes, that is correct but you're not understanding the point that was being made. If the FD causes the encounter to reset, for example by there simply being no other target for the encounter to engage then ending Feign Death will not automatically re-engage the encounter.

It never did on retail, ever. I cannot stress that enough. And if you claim otherwise, you never played retail. Ever.

Let's say you pull a few mobs with your Hunter, no pet or party members and you kite them for a while. Let's say, 300 yards or so, well outside of their original aggro range. Now, you FD and they run back to their original spawn locations. Now you end you FD. You really think that they will suddenly, magically run back from their spawn location to you? Because, that never happened on retail. Again, never, ever.

Now, an additional problem with Atlantiss' FD -- It ends on its own. I have seen that happen far too often now. I'll FD and just let go of all input peripherals. Magically, my character will just stand up again. Without having received damage from any source (direct or DoT). That is also very unBlizzlike behaviour.
